typescript/cloudwatch/evidently-client-side-evaluation-lambda/README.md

```diff
@@ -0,0 +1,38 @@
+# Evidently Client-side Evaluation Lambda
+<!--BEGIN STABILITY BANNER-->
+---
+
+![Stability: Stable](https://img.shields.io/badge/stability-Stable-success.svg?style=for-the-badge)
+
+> **This is a stable example. It should successfully build out of the box**
+>
+> This examples is built on Construct Libraries marked "Stable" and does not have any infrastructure prerequisites to build.
+
+---
+<!--END STABILITY BANNER-->
+
+This example creates a new lambda function that can call the Evidently EvaluateFeature API with minimal latency overhead
```

Comment:
Tell me

* What is Evidently
* What does this API do for me?
* How do I know this thing is working?

Tell me the utility!

I would heavily suggest linking to at least the marketing landing pages for the service you're demonstrating.

Reply:
Sure, will add docs